### DISCUSSION TITLE
What about supporting IDS messages for Broker communication?

### DISCUSSION THREAD

**Discussion by tomkxy** [eclipse-edc/Connector#623] [Q&A]
Looking at the code it seems to me that ids:ConnectorUpdateMessage and ids:ResourceUpdateMessage are not yet supported. Those would be required to register assets with an IDS broker. Am I right or did I overlook something?

**Comment by paullatzelsperger** [comment]
@tomkxy Your observation is correct, but as of now I know of no plans to do that. 
Interop with legacy systems in general and incorporating the IDS Broker in particular are not on any roadmap I am aware of. 

We will come up with a concept for a registry service of some sort in the future.

**Comment by mspiekermann** [comment]
@tomkxy : the idea is to have a look into IDSA RAM 4.0, which is currently worked on by IDSA. Since the EDC's clear goal is to be compliant to IDSA and Gaia-X specifications, providing these interfaces will be discussed and scheduled if we have more detailed information. @paullatzelsperger is refering to the concrete IDS Metadata Broker implementation, that can be found in the IDSA Github-Org. We see a lot of limitations and open issues that we need to work closely with the appropriate team to resolve first.
